src/App.js

```diff
@@ -2,12 +2,17 @@ import AppHeader from '~/containers/layout/AppHeader'
 import AppMain from '~/containers/layout/AppMain'
 import { ThemeProvider } from '@mui/material'
 import theme from './styles/custom-mui.styles'
+import Layout from '~/components/layout/Layout'
 
 const App = () => (
   <div className="App">
     <ThemeProvider theme={ theme }>
-      <AppHeader />
-      <AppMain />
+      <Layout>
+        { /*Router will be here*/ }
```

Comment:
or should i place Layout inside AppMain?

Reply:
Guest Layout, yeap in App Main